### ISSUE TITLE
Increase VRF configuration scale

### ISSUE THREAD

**Issue by onurgashi** [aristanetworks/ansible-avd#314]
**Issue Type**
- Template enhancement

**Is your feature request related to a problem? Please describe.**
Would like to have larger scale of VRF option, it is understood that 1024 is a limitation, however if we have multiple tenants shared across the Fabric, there should be an option to have more VRFs in total spread around for different tenants.

**Describe the solution you'd like**

The current 1024 is not enough, if we can either **double** the size or go a bit more beyond it that would be great.

**Describe alternatives you've considered**
None really. As many factors rely on this L3VNI/RD/RT

**Additional context**
Providing that there is space for scale taking into factor that not all VRFs will be in a single location.
The only limiting factor is the    

```
    # Define L3 network services organized by vrf.
    vrfs:
      # VRF name | Required
      < tenant_a_vrf_1 >:

        # VRF VNI | Required.
        # The VRF VNI range is limited.
        vrf_vni: <1-1024>
```
That many things use this to generate EVPN config.

**Comment by ClausHolbechArista** [comment]
I don't see a need to set a limit here. We don't limit VNI numbers anywhere else, so let it be up to the user to decide on scheme.
The EOS limit of 1024 concurrent VRFs should not be handled in a dataset that spans multiple devices and tenants.

**Comment by onurgashi** [comment]
Resolved with PR #449
